torch/csrc/nn/type_checks.h

```diff
@@ -28,7 +28,7 @@ static inline int get_device(PyObject* args) {
   for (int i = 0, n = PyTuple_GET_SIZE(args); i != n; i++) {
     PyObject* arg = PyTuple_GET_ITEM(args, i);
     if (THPVariable_Check(arg)) {
-      auto& tensor = THPVariable_UnpackData(arg);
+      auto& tensor = THPVariable_Unpack(arg);
```

Comment:
It's okay to use `THPVariable_Unpack()` instead of `THPVariable_UnpackData()` here, because the rest of the function doesn't perform any gradient-accumulating operations on `tensor`.